.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 
+logs/
```

Comment:
add line below and remove an extra line from above - these are standards while coding we have to follow.